Simplified/NYPLAccount.h

```diff
@@ -54,14 +54,8 @@ static NSString *const NYPLAccountLoginDidChangeNotification =
 
 - (BOOL)hasBarcodeAndPIN;
 
-- (BOOL)hasCredentials;
```

Comment:
Are these being removed because they are unused? If so, can we put them back? iirc this may break OpenEBooks merge.

Reply:
Yeah, they are unused. Are they still used by OpenEBooks or is it just to avoid merge conflicts?

Also, I was wondering about that merge: is the current open PR still ok to review? I was wondering what's the plan there.

Reply:
Please do review that open PR!

Reply:
I readied the methods still used by OpenEBooks.